handbook/handbook.company.values.md

```diff
@@ -88,16 +88,43 @@ Iteration means that we will hit the deadline but are flexible about the scope.
 
 We look for virtuous cycles and long term growth in all that we do. This means we think about both the long term **benefits and costs** of the actions we take and invest in areas that will take many years to bear fruit. It also means we don't do things as a one off and try to find ways to re-use existing work. 
 
+We leave things better than we found it. Code, knowledge, the world - we like to leave things so that the next person coming back do it will have a better way of starting.
 
 ### In Practice
-- We pay down technical debt by dedicating cycles to it every sprint
+- all important decisions and learnings are written down in the company handbook so everyone can benefit
+- We are collecting and indexing the worlds data in publically available vaults on [Personal Knowledge Management](https://pkm.dendron.so/), [AWS](https://aws.dendron.so/) and of course [XKCD](https://xkcd.dendron.so/)
+- we are not shy about prioritizing and will opt to scope down or postpone feature work that would otherwise force us to work unsustainably (long hours, taking on tech debt, buggy). 
+
+### Key Principles
```

Comment:
Swapped Key principles and In practice since it got hard to follow otherwise.

Reply:
updated line 96